package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/app",
   "typings": "dist/app",
   "scripts": {
-    "lint": "eslint . --max-warnings=0",
+    "lint": "eslint --ext .js,.ts .",
```

Comment:
Can we keep the `--max-warnings=0` argument, so we don't end up with warning messages in Github like those in [`src/lin/pod.ts`](https://github.com/alphagov/di-solid-prototype/pull/90/files#diff-694c4b46ce84c26f453de13b705eb61293bc76af65379702834bd6f72a1eb3bfL115).

We'll also need to add an exception or ignore for the console logging that's currently causing the warnings.

Reply:
Oh wow, hadn't spotted github was doing that - yeah good shout.

Reply:
@alex9smith done!
https://github.com/alphagov/di-solid-prototype/pull/90/commits/eb70b6fcb590bc8b3416e4c42f4d3fb4e58f4704